### ISSUE TITLE
kubernetes 1.11.0 kube-proxy error

### ISSUE THREAD

**Issue by jicki** [kubernetes/kubernetes#65611]
## kubernetes version 1.11.0

```
CentOS Linux release 7.4.1708 (Core) 


[root@kubernetes-65 ~]# uname -a
Linux kubernetes-65 4.4.111-1.el7.elrepo.x86_64 #1 SMP Wed Jan 10 13:12:02 EST 2018 x86_64 x86_64 x86_64 GNU/Linux


[root@kubernetes-65 ~]# rpm -qa|grep ipset
ipset-libs-6.29-1.el7.x86_64
ipset-6.29-1.el7.x86_64

```

* kube-proxy run  error  logs

```

6月 29 11:51:47 kubernetes-65 kube-proxy[73946]: E0629 11:51:47.439385   73946 ipset.go:156] Failed to make sure ip set: &{{KUBE-LOAD-BALANCER-SOURCE-IP hash:ip,port,ip inet 1024 65536 0-65535 Kubernetes service load balancer ip + port + source IP for packet filter purpose} map[] 0xc4203b6420} exist, error: error creating ipset KUBE-LOAD-BALANCER-SOURCE-IP, error: exit status 1
6月 29 11:51:52 kubernetes-65 kube-proxy[73946]: E0629 11:51:52.323873   73946 ipset.go:156] Failed to make sure ip set: &{{KUBE-EXTERNAL-IP hash:ip,port inet 1024 65536 0-65535 Kubernetes service external ip + port for masquerade and filter purpose} map[] 0xc4203b6420} exist, error: error creating ipset KUBE-EXTERNAL-IP, error: exit status 1
6月 29 11:51:57 kubernetes-65 kube-proxy[73946]: E0629 11:51:57.348685   73946 ipset.go:156] Failed to make sure ip set: &{{KUBE-LOAD-BALANCER hash:ip,port inet 1024 65536 0-65535 Kubernetes service lb portal} map[] 0xc4203b6420} exist, error: error creating ipset KUBE-LOAD-BALANCER, error: exit status 1
6月 29 11:52:02 kubernetes-65 kube-proxy[73946]: E0629 11:52:02.372216   73946 ipset.go:156] Failed to make sure ip set: &{{KUBE-LOOP-BACK hash:ip,port,ip inet 1024 65536 0-65535 Kubernetes endpoints dst ip:port, source ip for solving hairpin purpose} map[] 0xc4203b6420} exist, error: error creating ipset KUBE-LOOP-BACK, error: exit status 1
6月 29 11:52:07 kubernetes-65 kube-proxy[73946]: E0629 11:52:07.397256   73946 ipset.go:156] Failed to make sure ip set: &{{KUBE-LOOP-BACK hash:ip,port,ip inet 1024 65536 0-65535 Kubernetes endpoints dst ip:port, source ip for solving hairpin purpose} map[] 0xc4203b6420} exist, error: error creating ipset KUBE-LOOP-BACK, error: exit status 1


```

**Comment by liggitt** [comment]
duplicate of https://github.com/kubernetes/kubernetes/issues/65461?

**Comment by stewart-yu** [comment]
Thanks for your error catch. @jicki 
Please open the issuse with template, which convenience to others make choice about bug or not? thanks

**Comment by jicki** [comment]
@liggitt 

I test to #65461   

```
[root@kubernetes-65 ~]# ipset create foo hash:ip comment
ipset v6.29: Set cannot be created: set with the same name already exists
```

**Comment by m1093782566** [comment]
/close

It's fixed by #65533
